scanner/updater/version/RELEASE_VERSION

```diff
@@ -0,0 +1,7 @@
+#Version
```

Comment:
Can you expand this comment to document the purpose of this file, how it is consumed and supposed to be changed? Also, mention that it has to end with a new line, otherwise the last version will not be considered (according to your `jq` in the github actions steps).